services/src/pro/api/handlers/machine_learning.rs

```diff
@@ -59,14 +59,14 @@ pub(crate) async fn add_ml_model<C: ApplicationContext>(
     session: C::Session,
     app_ctx: web::Data<C>,
     model: web::Json<MlModel>,
-) -> Result<HttpResponse, MachineLearningError> {
+) -> Result<web::Json<MlModelNameResponse>, MachineLearningError> {
```

Comment:
add ModelNameReponse to utoipa response in the macro

Reply:
https://github.com/geo-engine/geoengine/pull/1007/commits/7955102e5a5049100a712e8fca5735846c6ad5ca